File: src/integTest/project/useScalaOnlyTest/app/src/androidTest/scala/jp/leafytree/android/libproject/app/Lib1JavaTest.scala
package jp.leafytree.android.libproject.app

import junit.framework.{TestCase, Assert}
import jp.leafytree.android.libproject.lib1.Lib1Java

class Lib1JavaTest extends TestCase {
  def test1() {
    Assert.assertEquals("Lib1Java", new Lib1Java().getName)
    Assert.assertEquals(List("Lib1Java").last, Option(new Lib1Java().getName).get)
  }
}

File: src/integTest/noScala/app/src/androidTest/java/jp/leafytree/android/libproject/app/Lib1JavaTest.java
package jp.leafytree.android.libproject.app;

import junit.framework.TestCase;

import jp.leafytree.android.libproject.lib1.Lib1Java;

public class Lib1JavaTest extends TestCase {
    public void test1() {
        assertEquals("Lib1Java", new Lib1Java().getName());
    }
}
